ckanext/depositar_theme/fanstatic/scss/_ckan.scss

```
@@ -0,0 +1,25 @@
+// CKAN Core CSS
```

```suggestion
// CKAN Core CSS
@use "sass:math";
@import "variables";
```
Align with the [upstream](https://github.com/ckan/ckan/blob/ckan-2.10.7/ckan/public/base/scss/_ckan.scss).